neuraxle/api/flask.py

```diff
@@ -59,7 +60,7 @@ def transform(self, data_inputs) -> Response:
         :return: flask response object
         """
         from flask import jsonify
-        return jsonify(self.encode(data_inputs))
+        return jsonify(self.encode(np.array(data_inputs).tolist()))
```

Comment:
This fix should not be done to make the test pass because we used a numpy encoder just before. Please investigate why this happens here. It may have to do with the conv fix in the Data Container.

Reply:
somehow, I didn't have to change anything. the test is passing don't know what happened. just removed the changes. fixed